App.js

```diff
@@ -322,7 +303,8 @@ export default class App extends React.Component {
                   name="Chat"
                   component={ChatScreen}
                   options={{headerShown: false}}
-                  initialParams={{ id: 1 }} />
+                  initialParams={{ id: 1 }} 
+                  getId={({ params }) => params.id} />
```

Comment:
Why this?

Reply:
This was just coming from Vivek's PR, but I see now it is having the problem with going back to the old chat.